nylas/client/restful_models.py

```diff
@@ -83,6 +93,12 @@ class Message(NylasAPIObject):
              "account_id", "object", "snippet", "starred", "subject",
              "thread_id", "to", "unread", "starred", "_folder", "_labels",
              "headers"]
+    datetime_attrs = {
+        "received_at": "date",
+        # The following are used for filtering:
```

Comment:
I was thinking instead of using `cls.datetime_attrs` as the argument to `convert_datetimes_to_timestamps` above we could use `cls.datetime_filter_attrs` instead since that's the only place we care about these filter attrs. And when defining the class field we could extend `cls.datetime_attrs` with the additional filter attrs.